Tests/Functional/Sources/ErrorHandlingTest.m

```diff
@@ -125,10 +126,10 @@ - (void)testDescriptionForSearchActionWithAssertion {
       onElementWithMatcher:grey_accessibilityLabel(@"Invalid Scroll View")]
       assertWithMatcher:grey_sufficientlyVisible()
                   error:&error];
-  NSString *searchAPIDescription = @"\"Search API Info\" : \"Search Action:";
```

Comment:
Can you show me how this is printed?

Reply:
We talked about this offline, but it contains whitespace padding of ~10 characters and should be removed, and I believe you are tracking that change already.